kobo/apps/accounts/models.py

```diff
@@ -1,8 +1,15 @@
 from allauth.account.signals import email_confirmed
+from django.contrib import admin
 from django.db import models
 from django.dispatch import receiver
 
 
+class EmailContent(models.Model):
+    email_name = models.CharField(max_length=120)
```

Comment:
email_name + section_name must be unique_together, right? There can't be two versions of the same content block on the same email. email_name is the template name, yes? It should have choices that allow for only valid choices. Perhaps section should be the same, but I think that's too tedious and would be fine leaving it free.

Reply:
They will be added in the next commits, I forewent them for now in favour of getting the prototype done as quickly as possible as asked